backend/inspirehep/editor/views.py

```diff
@@ -44,6 +45,16 @@ def revert_to_revision(endpoint, pid_value):
         raise EditorRevertToRevisionError
 
 
+@blueprint.route("/<endpoint>/<int:pid_value>", methods=["GET"])
+@login_required_with_roles([Roles.cataloger.value])
+def get_record_and_schema(endpoint, pid_value):
+    pid_type = PidStoreBase.get_pid_type_from_endpoint(endpoint)
+    record = InspireRecord.get_record_by_pid_value(pid_value, pid_type)
+    return jsonify(
+        {"record": {"metadata": record}, "schema": load_schema(record["$schema"])}
```

Comment:
we could also drop `metadata` right? Since we're changing the editor as well

Reply:
I just kepth the this if we need to add more info for the record later, like updated etc.